description: gcc requires complete definition when types are used in unique_ptr
author: Michael Gilbert <mgilbert@debian.org>

--- a/services/preferences/tracked/pref_hash_filter.h
+++ b/services/preferences/tracked/pref_hash_filter.h
@@ -22,8 +22,8 @@
 #include "services/preferences/tracked/hash_store_contents.h"
 #include "services/preferences/tracked/interceptable_pref_filter.h"
 #include "services/preferences/tracked/tracked_preference.h"
+#include "services/preferences/tracked/pref_hash_store.h"
 
-class PrefHashStore;
 class PrefService;
 
 namespace base {
--- a/components/policy/core/browser/browser_policy_connector_base.h
+++ b/components/policy/core/browser/browser_policy_connector_base.h
@@ -15,10 +15,10 @@
 #include "components/policy/core/common/schema.h"
 #include "components/policy/core/common/schema_registry.h"
 #include "components/policy/policy_export.h"
+#include "components/policy/core/common/configuration_policy_provider.h"
 
 namespace policy {
 
-class ConfigurationPolicyProvider;
 class PolicyService;
 class PolicyServiceImpl;
 
